src/tool/subcommands/api_cmd/test_snapshot.rs

```diff
@@ -201,14 +212,16 @@ mod tests {
             .trim()
             .split("\n")
             .filter_map(|n| {
-                Url::parse(
-                    format!(
-                        "https://forest-snapshots.fra1.cdn.digitaloceanspaces.com/rpc_test/{n}"
-                    )
-                    .as_str(),
-                )
-                .ok()
-                .map(|url| (n, url))
+                let base_url = format!(
+                    "https://forest-snapshots.fra1.cdn.digitaloceanspaces.com/rpc_test/{n}"
+                );
+                let mut url = Url::parse(&base_url).ok()?;
+                // Append a timestamp to the URL as a `cachebust` query parameter
```

Comment:
Let's not do this. Updating a snapshot should be prohibited to not break unit tests for other branches

Reply:
Removed.